README.md

```diff
@@ -156,27 +156,11 @@ on:
 permissions: read-all
       
 jobs:
-  # Generate ldflags dynamically.
-  # Optional: only needed for ldflags.
-  args:
-    runs-on: ubuntu-latest
-    outputs:
-      ldflags: ${{ steps.ldflags.outputs.value }}
-    steps:
-      - id: checkout
-        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.3.4
-        with:
-          fetch-depth: 0
-      - id: ldflags
-        run: |
-          echo "::set-output name=value::$(./scripts/version-ldflags)"
```

Comment:
It may be useful to provide an example how to pass ldflags. Note that the config file in the doc also uses the ldflags. Shall we provide 2 different examples instead?
1. Simple
2. Use of ldflags


The example with ldflags could have:
```
run: |
  GIT_VERSION=$(git describe --tags --always --dirty)
  GIT_HASH=$(git rev-parse HEAD)
  # https://reproducible-builds.org/docs/source-date-epoch/
  # https://mirrors.edge.kernel.org/pub/software/scm/git/docs/git-log.html
  SOURCE_DATE_EPOCH=$(git log --date=iso8601-strict -1 --pretty=%ct)
  GIT_TREESTATE=$(if git diff --quiet; then echo "clean"; else echo "dirty"; fi)
  PKG=$(go list -m | head -n1)/pkgname
  echo "-X $PKG.gitVersion=$GIT_VERSION -X $PKG.gitCommit=$GIT_HASH -X  $PKG.gitTreeState=$GIT_TREESTATE -X $PKG.buildDate=$SOURCE_DATE_EPOCH -w -extldflags \"-static\""

```

Reply:
Apologies, I'm New to Go. If setting ldflags is common practice, let's keep it in the example. I'll update this PR to remove the script and use something like the suggestion in your comment. Thanks.

Reply:
SG. Yes the common use case for ldflags is to set version, commit hash and time of latest commit automatically.